ansicolor_windows.go

```diff
@@ -78,6 +78,16 @@ const (
 	ansiBackgroundCyan    = "46"
 	ansiBackgroundWhite   = "47"
 	ansiBackgroundDefault = "49"
+
+	ansiLightForegroundGray    = "90"
+	ansiLightForegroundRed     = "91"
+	ansiLightForegroundGreen   = "92"
+	ansiLightForegroundYellow  = "93"
+	ansiLightForegroundBlue    = "94"
+	ansiLightForegroundMagenta = "95"
+	ansiLightForegroundCyan    = "96"
+	ansiLightForegroundWhite   = "97"
+	ansiLightForegroundDefault = "99"
```

Comment:
I refer to the following definition, but did not find the definition of 99.
Please you tell me the definition of I to 99.
http://en.wikipedia.org/wiki/ANSI_escape_code#CSI_codes

In addition to that, also I would like to add high-intensity background color, if possible.